### DISCUSSION TITLE
What is the expected `renovateVersion` of the hosted app?

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [renovatebot/renovate#22738] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Wanted end result.

To run the latest version of Renovate.

My hosted app jobs running today are still using `"renovateVersion": "35.110.0"`, which is 11 days and 66 commits behind the latest release.

I have never known the hosted app to run anything other than the latest release, but I can't find any information to explain why it's currently running an older release.

### What you tried so far.

Searching the documentation website and the GitHub repository for various combinations of "Renovate version", `renovateVersion`, hosted Renovate version, and so on. I haven't had much luck finding useful search terms, sorry if this information is available somewhere.

### Relevant debug logs

<details><summary>Logs</summary>

```
INFO: Repository started
{
  "renovateVersion": "35.110.0"
}
```

</details>

**Comment by rarkins** [comment, marked as answer]
The version in the hosted app is approved/updated manually, so it's tended to fall anything from a few hours behind to a week behind in the past. 

We'll be updating this one shortly and aiming to have it less in future.

**Comment by HonkingGoose** [comment]
@rarkins updates the hosted app manually. That way they can delay the rollout of a new version if there's a problem with one of our new versions.

So that's why the hosted app may lag behind the latest open source release.

---

Edit: we should definitely explain this in the docs. We're getting more questions like yours. @rarkins do you want me to create a draft PR for you to review/edit?

**Comment by HonkingGoose** [comment]
> Searching the documentation website and the GitHub repository for various combinations of "Renovate version", `renovateVersion`, hosted Renovate version, and so on. I haven't had much luck finding useful search terms, sorry if this information is available somewhere.

There was one useful bit of information in our FAQ, but it's easily missed when searching:

> If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.

I hope adding a new section to the FAQ makes it easier for users to understand why the hosted app may lag behind the open source version:

- https://github.com/renovatebot/renovate/pull/22816

**Reply by aaronadamsCA** [reply]
That's great! Even as a long-time user I didn't know hosted releases were manual and could fall several days behind; my recent Renovate PR wasn't behaving as expected, and I spent quite a bit of time reviewing the logs before I realized the change simply hadn't been released. Thank you!

**Reply by HonkingGoose** [reply]
If I knew nothing about the Renovate app update process, I would expect it to match the OSS release precisely, like you. More users asked about the Mend-hosted app release schedule lately. So that's a sign I should improve the docs. 😄 

Thanks for letting me know the docs were missing this information! ❤️

**Reply by rarkins** [reply]
I'm not disputing your assumptions, but there are very few hosted applications in the world which behave this way (instant update of a live production system based on a release of an open source dependency).

**Reply by HonkingGoose** [reply]
I stand corrected! 😄

**Reply by HonkingGoose** [reply]
Is this a correct summary then?

1. `main` and `v35` should always be in a release-ready state, so we have good test coverage, and only merge PRs where all tests are green
2. Semantic Release automatically creates a new release when relevant commits are pushed to the branch
3. Self-hosted users can use Renovate to update Renovate itself
4. `@rarkins` decides when a release-ready version goes in production on the hosted app
